betaflight/betaflight
# UBLOX Full Power mode + default STATIONARY model

### Changes:
- `gps_ublox_acquire_model` was changed to STATIONARY model because it locks faster than AIRBORNE_4G. I tested on 3 different GPSes, @ZzyzxTek also tested it. (bn880 30-40s for 3d lock) Dynamic models do help getting faster gps locks in my experience.
- I added an option called `gps_ublox_full_power` which sets the gps to full power mode. If I understand it correctly then it won't save any power by turning off stellite tracking thus it will search for as many satellites as it can. The extra power consumption is negligible compared to the motors but it helps getting more satellites (and faster lock).

On default I turned on gps_ublox_full_power but if it set to OFF then it uses default Balanced mode.
I asked Oscar Liang about this and he said full power mode helps.

### Testing suggestions:

- Type `gpspassthrough` into cli, disconnect from configurator and read the currently configured power mode (PMS/Power Management Setup tab in U-center)
- Power cycle the quad
- Change  `gps_ublox_full_power` to OFF and check in U-center if the power mode changed to Balanced.
